content/v2.markdown

```diff
@@ -57,3 +57,120 @@ curl -H 'X-DNSimple-Domain-Token: DOMAIN-TOKEN' \
 
 The [Authentication](/v2/authentication/) page provides more information about how to use them.
 
+## Responses
+
+All successful responses will be returned as a JSON object with at least one key: `data`.
+
+The `data` element will contain either a single JSON object or a list of JSON objects, depending on the endpoint.
+
+```
+{
+    "data": {
```

Comment:
In all the examples I've been using 2 chars indentation. Would you mind to adjust them?

I also suggest here and the `data` example below, to use just a small representation of the object rather a full one so that it will not distract the reader from the real focus.

```
{
  "data": [
    {
      "id": 1,
      "first_name": "Jane",
      "last_name": "Smith",
    },
    {
      "id": 3,
      "first_name": "John",
      "last_name": "Smith",
    }
  ],
  "pagination": {
    "current_page": 1,
    "per_page": 30,
    "total_entries": 2,
    "total_pages": 1
  }
}
```